### PR TITLE
Add all Buick vehicles to common.js

### LINKED ISSUE #44: Add all Buick vehicles to common.js
Find all Buick vehicles in Appendix A of this document https://www.scca.com/downloads/49336-2020-solo-rules-complete-reduced-2020-03-16/download and add them to https://github.com/Bjorn248/scca_classifier/blob/master/common.js in a similar way to other Makes.

### PR DESCRIPTION
**Pull request by dinukad**
resolve #44

### CODE REVIEW

**Review by Bjorn248 — CHANGES_REQUESTED** [review]
Please go through the Appendix to add more than just the street classes. For example the Buick Regal can also run in `esp`. Thanks.

**Comment by dinukad** [comment]
@Bjorn248 did some changes as requested. Please check.

File: common.js [hunk 1]
```diff
@@ -1197,6 +1197,51 @@ const allSoloCars = {
...
+      '2016': ['gs'],
+      '2017': ['gs'],
+      '2018': ['gs'],
+      '2019': ['gs'],
+      '2020': ['gs']
+    },
+    'Regal (V6 & V8, RWD)': {
+      '1980': ['esp'],
+      '1981': ['esp'],
+      '1982': ['esp'],
```
Comment (Bjorn248) [inline common.js:1219]: The 1982-1988 Regal can also run in `cp`

**Review by Bjorn248 — CHANGES_REQUESTED** [review]
Missing one last class, then this is ready to merge.

**Pull request merged**